app/models/embassy.rb

```diff
@@ -21,7 +21,7 @@ def remote_office
     elsif can_assist_in_other_location?
       remote_office = offices_in_remote_location.first
       RemoteOffice.new(name: remote_office.worldwide_organisation.name,
-                       location: remote_office.country,
+                       location: remote_office.country.name,
```

Comment:
Just a check since you mention in the commit message about the template doing this transformation - does this affect the existing rendering (i.e. are we going to be calling `.name` in a template now on a string)?

Reply:
Previously we were relying on the `to_s` method [defined by WorldLocation](https://github.com/alphagov/whitehall/blob/publish-embassies-index/app/models/world_location.rb#L61). So we've decided to call `name` explicitly here to populate the json. This doesn't affect the existing rendering.